src/controllers/beneficial.owner.gov.controller.ts

```diff
@@ -1,23 +1,42 @@
 import { NextFunction, Request, Response } from "express";
-import { BENEFICIAL_OWNER_GOV_PAGE, BENEFICIAL_OWNER_TYPE_URL } from "../config";
-import { getBeneficialOwnerGov, getBeneficialOwnerGovById, postBeneficialOwnerGov, removeBeneficialOwnerGov, updateBeneficialOwnerGov } from "../utils/beneficial.owner.gov";
+import {
+  getBeneficialOwnerGov,
+  getBeneficialOwnerGovById,
+  postBeneficialOwnerGov,
+  removeBeneficialOwnerGov,
+  updateBeneficialOwnerGov
+} from "../utils/beneficial.owner.gov";
+
+import * as config from "../config";
+
+import { isActiveFeature } from "../utils/feature.flag";
+import { getUrlWithParamsToPath } from "../utils/url";
 
 export const get = (req: Request, res: Response) => {
-  return getBeneficialOwnerGov(req, res, BENEFICIAL_OWNER_GOV_PAGE, BENEFICIAL_OWNER_TYPE_URL);
+  return getBeneficialOwnerGov(req, res, config.BENEFICIAL_OWNER_GOV_PAGE, config.BENEFICIAL_OWNER_TYPE_URL);
```

Comment:
Just wondering how this works without referencing BENEFICIAL_OWNER_TYPE_WITH_PARAMS_URL? Same for  getById in line 20.

Reply:
Well it works/worked because the fourth parameter being passed in (originally `config.BENEFICIAL_OWNER_TYPE_URL`) was used to set the 'Back' link and as that is another story I didn't bother changing those at this point.

However, I saw that @mattch1 did in his implementation for BO other and as it was such a simple change and to remain consistent I've now changed this here too.

Reply:
Ah, ok, if its back link related we probably should leave it until the back link story but no point reverting now for such a small change.